src/main/java/bobbybot/BobbyBot.java

```diff
@@ -0,0 +1,68 @@
+package bobbybot;
+
+import java.util.ArrayList;
+
+import bobbybot.commands.Command;
+import bobbybot.tasks.Task;
+import bobbybot.ui.Ui;
+
+/**
+ * BobbyBot is a chatbot that helps you keep track of your tasks.
+ */
+public class BobbyBot {
+    protected static final String NAME = "BobbyBot";
+    protected static final String FILE_PATH = "./data/bobbybot.txt";
+    protected TaskList tasks;
+    protected final Ui ui;
+    protected final Storage storage;
+
+    private ArrayList<Command> commandStack = new ArrayList<>();
+
+    /**
+     * Creates an instance of BobbyBot.
+     */
+    public BobbyBot() {
+        ui = new Ui();
+        storage = new Storage(FILE_PATH);
+        try {
```

Comment:
I think it would be better for you to extract the try catch statement into another function for better readability